### PR TITLE
buildable in closure save -> saveValue API change

### PR DESCRIPTION
**Pull request by e111077**
Please review DO NOT MERGE

This builds with closure and passes presubmit things

### CODE REVIEW

**Comment by e111077** [comment]
oops forgot to fix tests

File: app-indexeddb-mirror/app-indexeddb-mirror-worker.js [hunk 1]
```diff
@@ -240,8 +242,8 @@
       port.addEventListener('message', function(event) {
         this.handleClientMessage(event, port)
       }.bind(this));
-
-      if (!(port in this[CLIENT_PORTS])) {
+      let isPortInClient = port.toString() in this[CLIENT_PORTS];
```
Comment (cdata) [inline app-indexeddb-mirror/app-indexeddb-mirror-worker.js]: `let => var`

File: app-indexeddb-mirror/app-indexeddb-mirror-worker.js [hunk 2]
```diff
@@ -307,7 +309,7 @@
     }
   };
 
-  self.appIndexedDBMirrorWorker = new AppIndexedDBMirrorWorker();
+  self.appIndexedDBMirrorWorker = new AppIndexedDBMirrorWorker('', '');
```
Comment (cdata) [inline app-indexeddb-mirror/app-indexeddb-mirror-worker.js]: Can we just annotate the constructor to suggest that there are defaults set internally?

File: app-indexeddb-mirror/app-indexeddb-mirror.html [hunk 3]
```diff
@@ -181,7 +181,7 @@
            * Worker spawned from `workerUrl`.
            */
           client: {
-            type: Polymer.AppIndexedDBMirrorClient,
+            type: Object,
```
Comment (cdata) [inline app-indexeddb-mirror/app-indexeddb-mirror.html]: But why...? This is less accurate 😢

File: app-indexeddb-mirror/app-indexeddb-mirror.html [hunk 4]
```diff
@@ -253,7 +248,7 @@
             this.persistedData = this.data;
             this.linkPaths('data', 'persistedData');
           } else {
-            this.unlinkPaths('data', 'persistedData');
+            this.unlinkPaths('data');
```
Comment (cdata) [inline app-indexeddb-mirror/app-indexeddb-mirror.html]: 👍

File: app-indexeddb-mirror/common-worker-scope.js [hunk 5]
```diff
@@ -49,5 +49,5 @@
     }
   }.bind(this));
 
-  self.importScripts([workerScript]);
+  self.importScripts(workerScript);
```
Comment (cdata) [inline app-indexeddb-mirror/common-worker-scope.js]: It's a small (JavaScript) miracle that this ever worked...

File: app-indexeddb-mirror/common-worker.html [hunk 6]
```diff
@@ -95,10 +98,10 @@
        * A proxy method that forwards all calls to the backing `WebWorker`
        * instance.
        *
-       * @param {...} removeEventListenerArgs The arguments to call the same
+       * @param {Array} removeEventListenerArgs The arguments to call the same
```
Comment (cdata) [inline app-indexeddb-mirror/common-worker.html]: I don't think this is right.

File: app-indexeddb-mirror/app-indexeddb-mirror-worker.js (hunk 1 above)
Comment (e111077) [inline app-indexeddb-mirror/app-indexeddb-mirror-worker.js]: uhhhhhh

File: app-indexeddb-mirror/app-indexeddb-mirror.html (hunk 3 above)
Comment (e111077) [inline app-indexeddb-mirror/app-indexeddb-mirror.html]: It didn't like pointing to that. it kept saying that object was undefined

File: app-indexeddb-mirror/app-indexeddb-mirror-worker.js (hunk 2 above)
Comment (e111077) [inline app-indexeddb-mirror/app-indexeddb-mirror-worker.js]: yes we can

**Comment by e111077** [comment]
@cdata i got it!

File: app-indexeddb-mirror/common-worker.html [hunk 7]
```diff
@@ -98,7 +101,7 @@
        * @param {...} removeEventListenerArgs The arguments to call the same
        * method on the `WebWorker` with.
        */
-      removeEventListener: function() {
+      removeEventListener: function(removeEventListenerArgs) {
```
Comment (cdata) [inline app-indexeddb-mirror/common-worker.html]: Remove `removeEventListenerArgs`?

**Comment by e111077** [comment]
@cdata double checked and it builds just fine

**Comment by cdata** [comment]
LGTM 👍

**Pull request merged**